.github/workflows/bazel.yml

```diff
@@ -81,11 +81,11 @@ jobs:
       matrix:
         include:
           - bazel-config: ""
-            bazel-target: "//..."
```

Comment:
nitpick: Maybe call this `bazel-target-kind` or `target-kind` now?

Reply:
Sorry, only noticed this comment after merging. I can take this into another PR.